en/docs/ipaas/micro-integrator/develop-integrations-with-integration-studio.md

```diff
@@ -122,12 +122,17 @@ To configure logging in MI instances, you can use environment variables with spe
 2. Replace the dot character in the package name with an underscore.
 3. Set the variable value to the required logging level for the corresponding package or class.
 
-For example, to set the logging level for the `org.apache.synapse.transport.http.wire` package to `debug`, define the
-following environment variable:
-
-```
-export logging_level_org_apache_synapse_transport_http_wire=debug
-```
+For example, to enable wire logs logging level of the `org.apache.synapse.transport.http.wire` package needs to be changed
+to `debug`. To configure logging define the environment variable as follows:
+
+- Go to the `Deploy` page and go to the `Configs & Secrets` tab.
```

Comment:
Let's use numbering here.

Reply:
1. In the left navigation menu, click Deploy and then click **Configs & Secrets**.
2. Click **+ Create**.
3. Select `ConfigMap` as the **Config Type** and  `Environment Variables` as the **Mount Type**. 
4. Click **Next** to add the mount configurations.
5. Add the following follows to the mount configuration:
 <Add the values in a table form>